numpy/lib/function_base.py

```diff
@@ -1455,17 +1455,18 @@ def angle(z, deg=False):
     return a
 
 
-def _unwrap_dispatcher(p, discont=None, axis=None):
+def _unwrap_dispatcher(p, discont=None, axis=None, min_val=None, max_val=None):
```

Comment:
I think I'd like these to be keyword-only:
```suggestion
def _unwrap_dispatcher(p, discont=None, axis=None, *, min_val=None, max_val=None):
```